### DISCUSSION TITLE
Explain False Waypoints in FPL??

### DISCUSSION THREAD

**Discussion by DFWSupertrooper** [flybywiresim/aircraft#3743] [Q&A] [answered]
OK, so a better understanding of why and how MSFS and/or the A320 create false waypoints in the flight plan might help us all troubleshoot issues, so can someone who knows explain _why_ they are created and _how_ they are created and what the abbreviations mean (_CI###_, _D##_ as examples)?

For those who don't understand what I'm asking...when one creates a FPL using either MSFS World Map or SimBrief or a combination thereof, the waypoints are clear.  There is a SID for departure, STAR for arrival, and approach and they almost always match charts, but when one loads the A/C and checks the loaded FPL in the MCDU, there are always D's, and CI's, and a bunch of other stuff that often gets added between the known/accepted W/Ps.

For our education, can someone please explain the need for these, what they do, and how the sim creates them?

**Comment by hotshotp** [comment, marked as answer]
custom flight plan manager is in work right now and will be released some time soon. msfs flight plan is a mess XD

**Reply by DFWSupertrooper** [reply]
Are you saying the A/C is adding them (default Asobo code in the Airbus) and that when FBW gets done with the custom work this will no longer be the case?  Thereafter, we can expect the MCDU flight plan to _only_ use _actual_ waypoints based on charts??  I want to make sure I fully understand your point.

**Reply by hotshotp** [reply]
Yeah! So basically asobo flight plan manager is a mess - we are redoing everything and replacing it in the future. It will be very realistic and will not have any strange unnecessary waypoints :)

**Reply by DFWSupertrooper** [reply]
Awesome!  I like the flight _planning_ part - easy for me to match up to my SimBrief route and get ATC, I just don't care for how the fake waypoints get loaded after the plan is created, so it'll be nice to avoid those.  Most flights, they're not an issue but still not right.  Thx!

**Reply by robinottawa** [reply]
Mine seems to work fine. I thought the times when the globe planning tool WPs did not match what was in the FMS when loaded, it was my fault for not doing the globe thing right. But at least they were on the route and not some random Fs and Cs.
